demosplan/DemosPlanCoreBundle/ResourceTypes/OrgaResourceType.php

```diff
@@ -186,45 +200,41 @@ protected function getProperties(): array
-                return $orga->getHouseNumber();
-            }),
-            $this->createAttribute($this->submissionType)->readable(true, static function (Orga $orga): string {
-                return $orga->getSubmissionType();
-            }),
-            $this->createAttribute($this->types)->readable(true, function (Orga $orga): array {
-                return $orga->getTypes($this->globalConfig->getSubdomain());
-            }),
-            $this->createAttribute($this->registrationStatuses)->readable(true, [$this, 'getRegistrationStatuses']),
+            $this->createAttribute($this->state)
+                ->readable(true, static fn (Orga $orga): string => $orga->getState()),
+            $this->createAttribute($this->street)
+                ->readable(true, static fn (Orga $orga): string => $orga->getStreet()),
+            $this->createAttribute($this->houseNumber)
+                ->readable(true, static fn (Orga $orga): string => $orga->getHouseNumber()),
+            $this->createAttribute($this->submissionType)
+                ->readable(true, static fn (Orga $orga): string => $orga->getSubmissionType()),
+            $this->createAttribute($this->types)
+                ->readable(true, fn (Orga $orga): array => $orga->getTypes($this->globalConfig->getSubdomain())),
+            $this->createAttribute($this->registrationStatuses)
+                ->readable(true, [$this, 'getRegistrationStatuses']),
             $this->createToOneRelationship($this->currentSlug, true)->readable(true),
-            $this->createToManyRelationship($this->customers)->readable(false, static function (Orga $orga): Collection {
-                return $orga->getCustomers();
-            }),
-            $this->createToManyRelationship($this->departments)->readable(false, static function (Orga $orga): TightencoCollection {
-                return $orga->getDepartments();
-            }),
-            $this->createAttribute($this->isPlanningOrganisation)->readable(true,
+            $this->createToManyRelationship($this->customers)
+                ->readable(false, static fn (Orga $orga): Collection => $orga->getCustomers()),
+            $this->createToManyRelationship($this->departments)
+                ->readable(false, static fn (Orga $orga): TightencoCollection => $orga->getDepartments()),
+            $this->createAttribute($this->isPlanningOrganisation)->readable(
+                true,
                 function (Orga $orga): bool {
-                    return $orga->hasType(OrgaType::MUNICIPALITY, $this->globalConfig->getSubdomain())
-                        || $orga->hasType(OrgaType::PLANNING_AGENCY, $this->globalConfig->getSubdomain())
-                        || $orga->hasType(OrgaType::HEARING_AUTHORITY_AGENCY, $this->globalConfig->getSubdomain());
+                    $currentSubdomain = $this->currentContext->getCurrentCustomer()->getSubdomain();
```

Comment:
The missing method isn't defined in the interface but in the implementation of  Customer.php. Therefore this is actually working, but still incorrect because of the missing definition in CustomerInterface.php.

Reply:
https://github.com/demos-europe/demosplan-addon/pull/44